### DISCUSSION TITLE
Physics explanation behind the DPK dose profile with different voxel sizes

### DISCUSSION THREAD

**Discussion by ShahinGHD** [nrc-cnrc/EGSnrc#991] [Questions]
Hi everyone, I am simulating a simple geometry to get the DPK profiles of the Lu-177
i ran 4 different simulations with the same histories and geometry, the difference is only with the voxel sizes, I have these:
first = 2x2x2 mm voxel size
second = 1x1x1 mm voxel size
third = 0.5x0.5x0.5 mm voxel size
fourth = 0.1x0.1x0.1 mm voxel size
i multiplied the simulation result with the voxel masses to get the dose in "Gy"
so I am pretty sure the voxel masses are taken into account.

however, I am getting different DPK profiles, the bigger voxel size has higher DPK profile than the 0.1 mm, what is the reason here?

Here is the picture of my DPK profiles

[DPK profiles](https://pasteboard.co/04VQ6bF4OthH.png)

**Comment by rtownson** [comment]
Just to be sure, you said multiplied but I think you mean that you divided the energy deposited by the mass of the voxel, to get the Gy, right?

The next thing to check is the uncertainty - it will be much, much larger for the smaller voxel sizes if you used the same number of histories. That's why results are generally normalized per number of histories, so that it's independent and you can run as many histories as are necessary to get a good uncertainty.

**Reply by ShahinGHD** [reply]
Thanks for the reply, Well, the S values are in Gy/Bq.s. As DPK is only one Bq.s we assume the answer is Gy already. Since Gy is J/KG  mass, if you multiply it by mass you will get the Jules. I mistyped in the plot its jules not Gy.

The uncertainty is under 0.07% for all since I used 10^10 histories. it seems its just how the EGS works, when I sum all the energies inside all the voxels in 3D, they agree with each other. the dose profiles in jules are not.

**Reply by rtownson** [reply]
The larger voxels are expected to have a larger energy deposited in J, because they have a larger volume so more particles deposited energy inside of them.
